app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable that tracks all instances of `Animal`. Ensure that this list is managed correctly to avoid memory leaks or incorrect state tracking.